.github/workflows/build.yml

```diff
@@ -70,11 +70,6 @@ jobs:
             - run: cmake --build build
             - run: ctest --output-on-failure --test-dir build/
```

Comment:
FYI @ncesario-lunarg who is looking to do similar work in glslang.

`ctest` now handles the find_package integration testing. Making it require little to no CI effort.